packages/maskbook/src/components/InjectedComponents/ToolboxHint.tsx

```diff
@@ -133,11 +133,15 @@ export function ToolboxHint(props: ToolboxHintProps) {
     const { openDialog: openSelectProviderDialog } = useRemoteControlledDialog(
         WalletMessages.events.selectProviderDialogUpdated,
     )
+    const { openDialog: openCreateImportDialog } = useRemoteControlledDialog(
+        WalletMessages.events.createImportWalletDialogUpdated,
+    )
     const openWallet = useCallback(() => {
         if (account) {
             openSelectWalletDialog()
         } else {
-            openSelectProviderDialog()
+            // openSelectProviderDialog()
```

Comment:
```suggestion
```

Reply:
Fixed in https://github.com/DimensionDev/Maskbook/pull/3298/commits/ceb17bf35a934b93a06b2c82020d6bb7e973e963